nixos/modules/services/networking/wakeonlan.nix

```diff
@@ -31,6 +31,21 @@ in
 
     services.wakeonlan.interfaces = mkOption {
       default = [ ];
+      type = types.listOf (types.submodule { options = {
+        interface = mkOption {
```

Comment:
I think this should be converted to attrsOf, since specifying the same interface multiple times will have unintended consequences.

Reply:
you mean we should convert the list into an attr set? Or convert the submodule to an attr set?